lib/bowling.rb

```diff
@@ -0,0 +1,78 @@
+class Bowling
+  attr_reader :bonus, :total, :frame, :current_frame
```

Comment:
Why do you need to expose all of these instance variables? Could some stay private?

Reply:
Completely valid, only total and perhaps frame and current frame would be relevant to the user!